force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,12 +352,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
         // Loop through the list of words
+        for (Integer i = 0; i < words.size(); i++) {
+           if (words[i] == wordToFind) { 
+            return i;
+           } 
+           }
+           return -1;
```

Comment:
Try using the break keyword in this solution and try to line up the curly braces